code/exercises/affine-cipher/example.scm

```diff
@@ -0,0 +1,74 @@
+(import (rnrs))
+
+
+(define (encode key text)
+  ((get-translator key) (string-downcase text) 'encode))
+
+(define (decode key text)
+  ((get-translator key) text 'decode))
+
+;;; Private
+
+(define (atbash-format lst)
+  (let* ((chunks (windows lst 5))
+         (l (cons (car chunks)
+                  (map (lambda (chars) (cons #\space chars))
+                       (cdr chunks)))))
+    (list->string (fold-left append '() l))))
+
+(define (get-translator key)
+  (let ((a (car key))
+        (b (cdr key)))
+  (when (not (coprime? a 26))
+    (error 'affine-cipher "a and m must be coprime" a))
```

Comment:
Moved this here from `translator-factory` as it makes more sense here now I think.